allennlp/data/token_indexers/token_characters_indexer.py

```diff
@@ -1,113 +1,119 @@
-from typing import Dict, List
```

Comment:
I think you ended up changing the line endings on this file to DOS line endings.  Can you switch it back?